src/formatter.ts

```diff
@@ -0,0 +1,130 @@
+import { dim, magenta, red, reset, underline, yellow } from 'chalk';
+import { ESLint } from 'eslint';
+import stripAnsi from 'strip-ansi';
+import table from 'text-table';
+
+function pluralize(word: string, count: number): string {
+  return count === 1 ? word : `${word}s`;
+}
+
+export function formatter(
+  results: ESLint.LintResult[],
+  shouldIncludeTodo?: boolean
+) {
+  let output = '\n',
+    errorCount = 0,
+    warningCount = 0,
+    todoCount = 0,
+    fixableErrorCount = 0,
+    fixableWarningCount = 0,
+    chalkColorFunction = yellow,
+    hasAnyErrors = false;
```

Comment:
Done. FYI It was like this because this file is actually a copy with modifications of the [stylish formatter in ESLint](https://github.com/eslint/eslint/blob/aab1b840f9cffb2a76a5c9fe1852961be71dc184/lib/cli-engine/formatters/stylish.js), which is the default formatter.